crates/driver/example.toml

```diff
@@ -45,6 +45,19 @@ base-tokens = [
     "0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB",
     "0x6B175474E89094C44Da98b954EedeAC495271d0F",
 ]
+
+[[order-priority-strategy]]
+creation-timestamp = {}
```

Comment:
this seems like a very ugly way to configure this (the empty {} everywhere). Can we use something like this instead (may require tagging the strategy enum):

```toml
[[order-priority]]
strategy = "creation-timestamp"

[[order-priority]]
strategy = "order-class"

[[order-priority]]
strategy = "own-quotes"
max-order-age = "1m"
```

Reply:
Thanks for the tip. Refactored.